### PR TITLE
Task1 display

### PR DESCRIPTION
**Pull request by Noctiscreo**
Made the display store submissions in db.

### CODE REVIEW

**Review by Incorrect-password — APPROVED** [review]

File: collector-display.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+
+$db = new PDO('mysql:host=db;dbname=monsters', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->query('SELECT `Name`, `Weapon`, `Challenge` FROM `monsters`');
+$monsters = $query->fetchALL();
```
Comment (sjr-mayden) [inline collector-display.php]: Replace `fetchALL` with `fetchAll`

File: add-monster.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (sjr-mayden) [inline add-monster.php]: Cut this file, paste it somewhere safe (Desktop or wherever) add and commit the fact this was deleted

File: collector-display.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="UTF-8">
+    <title>Collector</title>
+    <link href="normalize.css" rel="stylesheet" type="text/css">
+    <link href="boilerplate.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width= device-width, initial-scale=1">
+</head>
+
+<body>
+
+<form method="post" action="add-monster.php">
```
Comment (sjr-mayden) [inline collector-display.php]: Cut the lines for the form and keep them somewhere safe, then add and commit this file

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: collector-display.php (hunk 1 above)
Comment (Noctiscreo) [reply collector-display.php]: Corrected fetchAll

File: add-monster.php (hunk 2 above)
Comment (Noctiscreo) [reply add-monster.php]: Cut it and add(ed) commit(ed).

**Review by sjr-mayden — APPROVED** [review]

**Pull request closed without merging**